scripts/network-init

```diff
@@ -28,13 +28,31 @@ prepare_networking() {
 
     if [ -e ${MANAGEMENT_CONF} ] ; then
         source ${MANAGEMENT_CONF}
-        if PHYSICAL_PIF="$($XE pif-list host-uuid=${INSTALLATION_UUID} device=${LABEL} params=uuid --minimal)" ; then
+        if [ ! -z "${BOND_MODE}" ] ; then
+            BOND_NETWORK="$($XE network-create name-label='Bonded pool-wide network with mode '${BOND_MODE})"
+            readarray -td, a <<<${BOND_MEMBERS}, ; unset 'a[-1]' ;
+            for MEMBER_DEVICE in "${a[@]}"
+            do :
+                if MEMBER_PIF="$($XE pif-list host-uuid=${INSTALLATION_UUID} device=${MEMBER_DEVICE} params=uuid --minimal)" ; then
+                    if [ -z ${PIFS} ] ; then
```

Comment:
This works by accident: when PIFS is empty, this is `[ -z ]` but should be `[ -z "" ]`.

Reply:
Indeed, Thanks!